demos/cryptoapp/app/src/main/java/com/guru/composecookbook/cryptoapp/ui/composewear/AndroidWearActivity.kt

```diff
@@ -0,0 +1,211 @@
+package com.guru.composecookbook.cryptoapp.ui
+
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.BlendMode.Companion.Screen
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.lifecycle.viewmodel.compose.viewModel
+import androidx.navigation.NavType
+import androidx.navigation.Navigator
+import androidx.navigation.navArgument
+import androidx.paging.PagingData
+import androidx.paging.compose.LazyPagingItems
+import androidx.paging.compose.collectAsLazyPagingItems
+import androidx.wear.compose.material.*
+import androidx.wear.compose.navigation.SwipeDismissableNavHost
+import androidx.wear.compose.navigation.composable
+import androidx.wear.compose.navigation.rememberSwipeDismissableNavController
+import coil.compose.rememberImagePainter
+import com.guru.composecookbook.charts.LineChart
+import com.guru.composecookbook.cryptoapp.data.db.models.Crypto
+import com.guru.composecookbook.cryptoapp.ui.home.CryptoHomeViewModel
+import com.guru.composecookbook.cryptoapp.ui.home.CryptoHomeViewModelFactory
+import com.guru.composecookbook.cryptoapp.ui.internal.extensions.roundToThreeDecimals
+import com.guru.composecookbook.cryptoapp.ui.internal.extensions.roundToTwoDecimals
+import com.guru.composecookbook.data.DemoDataProvider
+import com.guru.composecookbook.theme.ComposeCookBookTheme
+import com.guru.composecookbook.theme.gradientGreenColors
+import com.guru.composecookbook.theme.gradientRedColors
+import com.guru.composecookbook.theme.green500
+
+const val SYMBOL_ID = "symbolId"
+class AndroidWearActivity : ComponentActivity() {
+
+    @ExperimentalWearMaterialApi
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContent {
+            ComposeCookBookTheme(true) {
```

Comment:
This should be a androidx.wear.compose.material.MaterialTheme, not a androidx.compose.material.MaterialTheme

Reply:
Yes thanks for highlighting I will create new WearTheme and a separate module as well 👍🏽